Vagrantfile

```diff
@@ -98,9 +102,13 @@ if File.exist?(env_specific_config_path)
 end
 
 # Display the main current configuration parameters
-@ui.info "Welcome to Kubernetes playground!"
+@ui.info "Welcome to Kubernetes playground, version " + K8S_PLAYGROUND_VERSION
 @ui.info "Vagrant provider : " + settings["conf"]["vagrant_provider"]
 @ui.info "Networking plugin : " + settings["ansible"]["group_vars"]["all"]["kubernetes_network_plugin"]
+if ENV['VAGRANT_LOG']=='debug' or ENV['VAGRANT_LOG']=='info'
+  @ui.info "Active settings (from defaults.yaml and env.yaml):"
+  @ui.info settings
```

Comment:
Print settings.to_yaml if you want a nicely formatted YAML

Reply:
this is working very well, thank you ! this is the output of the info
I'm pushing a new commit 
```
---
conf:
  additional_ansible_arguments: ''
  additional_disk_size: 10240
  base_box_builder_mem: 2048
  master_mem: 2048
  minion_mem: 1024
  playground_name: k8s-p9
  base_box_builder_name: base-box-builder
  master_name: k8s-master-1
  minion_1_name: k8s-minion-1
  minion_2_name: k8s-minion-2
  minion_3_name: k8s-minion-3
  vagrant_provider: virtualbox
  kubernetes_nodes_base_box_id:
    virtualbox: bento/centos-7.4
    libvirt: centos/7
net:
  network_prefix: 192.168.0.
  network_prefix_ipv6: 'fde4:8dba:82e1:'
  subnet_mask: 255.255.255.0
  subnet_mask_ipv6: '56'
  master_ipv6_part: 'c40A::'
  minion_1_ipv6_part: 'c41e::'
  minion_2_ipv6_part: 'c41f::'
  minion_3_ipv6_part: 'c420::'
  libvirt_management_network_address: 192.168.121.0/24
pod_network:
  cluster_ip_cidr: 10.244.0.0/16
  service_ip_cidr: 10.96.0.0/12
ansible:
  group_vars:
    all:
      kubernetes_network_plugin: calico
      kubernetes_helm_mirror: https://get.helm.sh
      kubernetes_helm_ver: v2.16.3
      kubernetes_helm_os: linux
      kubernetes_helm_arch: amd64
      kubernetes_helm_archive_type: tar.gz
      kubernetes_helm_bin_dir: "/usr/local/bin"
      kubernetes_helm_checksums:
        v2.16.3:
          linux-amd64: sha256:9678eb726d6870e8eded204190357a0f494ed9d1803781b4bb80dde6427b086e
        v3.1.2:
          linux-amd64: sha256:e6be589df85076108c33e12e60cfb85dcd82c5d756a6f6ebc8de0ee505c9fd4c
    kubernetes-masters:
    kubernetes-minions:
```